static/templates/previewable_url_tooltip.hbs

```diff
@@ -0,0 +1,33 @@
+<style>
+    .previewable-container {
+        display: flex;
+    }
+    .previewable-container > img {
+        padding: 2px 5px 0 0;
+        width: 1rem;
+        height: 1rem;
+    }
+    .previewable-container > div {
+        min-width: 0;
+    }
+    .previewable-title, .previewable-details {
+        display: block;
+        max-width: 100%;
+        overflow: hidden;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+    }
+    .previewable-title {
+        font-size:medium;
+    }
+    .previewable-details {
+        color: gray;
+    }
+</style>
+<div class="previewable-container">
+    <img src="/static/images/{{ icon_path }}"/>
+    <div>
+        <div class="previewable-title">{{ title }}</div>
+        <span class="previewable-details">{{ owner }}/{{ repo }}#{{ issue_number }} opened by {{ user }}</span>
```

Comment:
These strings need to be tagged for translation.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html to learn how to do this correctly... though I think probably we just want to construct the strings inside JavaScript using `$t`, since that'll provide a more flexible interface if we want to support things that are not GitHub/GitLab in the future. So I'd just make the parameters be `hover_preview_icon_path`, `hover_preview_title`, `hover_preview_details`.